app/models/host/managed.rb

```diff
@@ -63,6 +63,13 @@ def build_hooks
   end
 
   include HostCommon
+  attr_accessible :build, :certname, :compute_attributes, :disk, :global_status,
+    :grub_pass, :interfaces_attributes, :installed_at, :image_id, :image_name,
```

Comment:
`interfaces_attributes` should be on Base as that's where the accepts_nested definition is.